docs/70-jobs.md

```diff
@@ -1,175 +1,175 @@
-
-### Commands
-Submit a Spark Job:
-
-```sh
-aztk spark job submit --id <your_job_id> --configuration </path/to/job.yaml>
-```
-
-NOTE: The Job id (`--id`) can only contain alphanumeric characters including hyphens and underscores, and cannot contain more than 64 characters. Each Job **must** have a unique id.
-
-#### Low priority nodes
-You can create your Job with [low-priority](https://docs.microsoft.com/en-us/azure/batch/batch-low-pri-vms) VMs at an 80% discount by using `--size-low-pri` instead of `--size`. Note that these are great for experimental use, but can be taken away at any time. We recommend against this option when doing long running jobs or for critical workloads.
-
-
-### Listing Jobs
-You can list all Jobs currently running in your account by running
-
-```sh
-aztk spark job list
-```
-
-
-### Viewing a Job
-To view details about a particular Job, run:
-
-```sh
-aztk spark job get --id <your_job_id>
-```
-
-For example here Job 'pipy' has 2 applications which have already completed.
-
-```sh
-Job             pipy
-------------------------------------------
-State:                              | completed
-Transition Time:                    | 21:29PM 11/12/17
-
-Applications                        | State          | Transition Time
-------------------------------------|----------------|-----------------
-pipy100                             | completed      | 21:25PM 11/12/17
-pipy200                             | completed      | 21:24PM 11/12/17
-```
-
-
-### Deleting a Job
-To delete a Job run:
-
-```sh
-aztk spark job delete --id <your_job_id>
-```
-Deleting a Job also permanently deletes any data or logs associated with that cluster. If you wish to persist this data, use the `--keep-logs` flag.
-
-__You are only charged for the job while it is active, Jobs handle provisioning and destorying infrastructure, so you are only charged for the time that your applications are running.__
-
-
-### Stopping a Job
-To stop a Job run:
-
-```sh
-aztk spark job stop --id <your_job_id>
-```
-Stopping a Job will end any currently running Applications and will prevent any new Applications from running.
-
-
-### Get information about a Job's Application
-To get information about a Job's Application:
-
-```sh
-aztk spark job get-app --id <your_job_id> --name <your_application_name>
-```
-
-
-### Getting a Job's Application's log
-To get a job's application logs:
-
-```sh
-aztk spark job get-app-logs --id <your_job_id> --name <your_application_name>
-```
-
-
-### Stopping a Job's Application
-To stop an application that is running or going to run on a Job:
-
-```sh
-aztk spark job stop-app --id <your_job_id> --name <your_application_name>
-```
+# Jobs
+In the Azure Distributed Data Engineering Toolkit, a Job is a serverless entity that runs applications and records application output. A Job will manage the full lifecycle of the infrastructure so you do not have to. This document describes how to create and use AZTK Jobs.
```

Comment:
"a Job is a serverless entity that..." That is not exactly true, right? Perhaps we should say "a Job is an entity that can run against a cluster that is automatically manged by the service. It can run applications and records outputs."